examples/pytorch/rgcn/experimental/README.md

```diff
@@ -75,90 +77,55 @@ python ~/dgl/tools/copy_files.py --ip_config ip_config.txt \
 
 ```bash
-pm_dglpart mag 2
-```
-This partitions the graph into two parts with a single process.
-
-```
-mpirun -np 4 pm_dglpart mag 2
+172.31.0.1
+172.31.0.2
+172.31.0.3
+172.31.0.4
 ```
-This partitions the graph into eight parts with four processes.
 
-```
-mpirun --hostfile hostfile -np 4 pm_dglpart mag 2
-```
-This partitions the graph into eight parts with four processes on multiple machines.
-`hostfile` specifies the IPs of the machines; one line for a machine. The input files
-should reside in the machine where the command line runs. Each process will write
-the partitions to files in the local machine. For simplicity, we recommend users to
-write the files on NFS.
+Users need to make sure that the master node (node-0) has right permission to ssh to all the other nodes.
 
-### Step 3: Convert the ParMETIS partitions into DGLGraph
+### Step 2: partition the graph.
 
-DGL provides a tool called `convert_partition.py` to load one partition at a time and convert it into a DGLGraph
-and save it into a file.
+The example provides a script to partition some builtin graphs such as ogbn-mag graph.
+If we want to train RGCN on 4 machines, we need to partition the graph into 4 parts.
 
+In this example, we partition the ogbn-mag graph into 4 parts with Metis. The partitions are balanced with respect to
+the number of nodes, the number of edges and the number of labelled nodes.
 ```bash
-python3 ~/dgl/tools/convert_partition.py --input-dir . --graph-name mag --schema mag.json --num-parts 2 --num-node-weights 4 --output outputs
+python3 partition_graph.py --dataset ogbn-mag --num_parts 4 --balance_train --balance_edges
 ```
 
-### Step 4: Read node data and edge data for each partition
+### Step 3: Launch distributed jobs
 
-This shows an example of reading node data and edge data of each partition and saving them into files located in the same directory as the DGLGraph file.
+DGL provides a script to launch the training job in the cluster. `part_config` and `ip_config`
+specify relative paths to the path of the workspace.
 
 ```bash
-python3 get_mag_data.py
+python3 ~/dgl/tools/launch.py \
+--workspace ~/rgcn/ \
+--num_trainers 1 \
+--num_servers 1 \
+--num_samplers 4 \
+--part_config data/ogbn-mag.json \
+--ip_config ip_config.txt \
+"python3 dgl_code/entity_classify_dist.py --graph-name ogbn-mag --dataset ogbn-mag --fanout='25,25' --batch-size 512  --n-hidden 64 --lr 0.01 --eval-batch-size 16  --low-mem --dropout 0.5 --use-self-loop --n-bases 2 --n-epochs 3 --layer-norm --ip-config ip_config.txt  --num-workers 4 --num-servers 1 --sparse-embedding  --sparse-lr 0.06 --node-feats"
 ```
 
-### Step 5: Verify the partition result (Optional)
-
-```bash
-python3 verify_mag_partitions.py 
+We can get the performance score at the second epoch:
+```
+Val Acc 0.4323, Test Acc 0.4255, time: 128.0379
```

Comment:
the original README shows how to partition with ParMETIS. can you copy the instructions of using ParMETIS?

Reply:
Added